crates/extensions/tedge_script_ext/src/lib.rs

```diff
@@ -13,6 +14,7 @@ pub struct ScriptActor;
 pub struct Execute {
     pub command: String,
     pub args: Vec<String>,
+    pub timeouts: Option<(Duration, Duration)>,
```

Comment:
I started with two options, but changed my mind because the interruption logic needs both. So if one in missing we need to use a default value. I prefer to push that (small) burden to the constructor rather than to the main logic.

In practice, one will only use `with_graceful_timeout`, the end user having to provide a single timeout (I gives 5 minutes for this script to run) and thin-edge internally adding an extra delay before sending a SIGKILL.